WooCommerce/Classes/ViewRelated/Dashboard/MyStore/PeriodDataViewController.swift

```diff
@@ -38,8 +38,27 @@ class PeriodDataViewController: UIViewController {
     @IBOutlet private weak var chartAccessibilityView: UIView!
 
     private var lastUpdatedDate: Date?
-    private var yAxisMinimum: String = Constants.chartYAxisMinimum.humanReadableString()
-    private var yAxisMaximum: String = ""
+
+    private var revenueItems: [Double] {
+        return orderStats?.items?.map({ $0.grossSales }) ?? []
```

Comment:
It's probably better to move the default value early to give more emphasis on what's important: mapping of `grossSales`. 

```suggestion
        (orderStats?.items ?? []).map(\.grossSales)
```